README.md

```diff
@@ -1,2 +1,109 @@
+
+⚫Large events like conferences etc where it can be difficult to keep track of who has attended and who didn't. 
+
+⚫Verifying employee showing up at meetings, training sessions etc.
+
+Project was firstly developed during [XRPL NFT hackathon.](https://devpost.com/software/xrp-nft-attendence)
+
+## Features
+
+⚫ Creating new events and batch minting NFTs for it
+
+⚫ Uploading NFT metadata to IPFS
+
+⚫ Checking if it's possible to claim NFTs for particular event
+
+⚫ Requesting claim for NFT from particular event. The new sell offer for NFT that has to be accepted by the user is returned
+
+⚫ Event attendees lookup
+
+⚫ NFT ownership verification
+
+## Documentation
+
+### Attendify.js
+
+⚫ Current version of documentation for Attendify.js is in the `docs` folder and was generated by jsdoc. Latest version of docs is hosted [here](https://justanotherdevv.github.io/AttendifyXRPL/Attendify.html).
+
+### REST API endpoints
+
+This API is built using the ExpressJS framework and the XRPL library, it connect to the local Attendify.js class to perform most of the actions. 
+
+⚫ `GET /api/mint` - Creates new claimable event and premints NFTs for it. You must have at least X XRP in order to use this to handle the reserve requirements for the NFTs and the minting process which uses [Tickets](https://xrpl.org/tickets.html), where X is amount of tokens that should be minted multiplied by 2 + additional XRP for feees.
+
+**@param {string} walletAddress** - Wallet address from user requesting claim
+
+**@param {integer} tokenCount** - Amount of NFTs that should be create for the event
+
+**@param {string} url** - Url to Image for event | Preferably this should be stored on IPFS
+
+**@param {string} title** - Title for event
+
+**@param {string} desc** - Description of event
+
+**@param {string} loc** - Location of event
+
+⚫ `GET /api/claim` - Checks if user is eligible for NFT claim or creates new offer that allows for claiming NFT for specific event. Endpoint returns response with sellOfferID that has to be accepted by user afterwards.
+
+**@param {string} walletAddress** - The wallet address of the user trying to claim
+
+**@param {integer} type** - The type of claim (1 for checking claim status and metadata, 2 for claiming)
+
+**@param {string} minter** - The minter address of the event
+
+**@param {string} eventId** - The ID of the event
+
+**Possible responses**
+
+    -`404` - Event with specified ID does not exist
+
+    -`claimed` - The NFT for specified event was already claimed by this user
+
+    -`empty` - All NFTs for this particular event were already claimed
+
+    -`success` - If `onlyCheckStatus` parameter was true it indicated that current parameters for selected event were sent
+
+    -`transferred` - Indicates that new sell offer for NFT related to selected event was created successfully and details were sent to user
+
+⚫ `GET /api/verifyOwnership` - Verifies whether or not user owns NFT with provided id for particular user.
```

Comment:
When I try this with: http://localhost:4000/api/verifyOwnership?walletAddress=rwbq1nusPZmRoQCSgwjNXGch8pJyhBJqEs&signature=1200037321ED62F3992FE1E78FF2FF4A2A908F6B154B646298435A0A0018929A440BDA976A4E744068BA83C4525F7122DEB627DB0CF5EBE86A254E226D763044C69EF27C412F3C67C426159631C5DCA27C863F5D76830D02140D2C53F9C8BC6BE0724D8904C1FA0B8114695010606A0E859D8B4E4D4F41F57E5C0BD389AD&minter=rMBU1wwAJZZz36YBNxug7YYHKeC5k3vqfG&eventId=0

(which is a wallet that has just successfully done the `api/claim` call, with a signed `AccountSet` transaction that does nothing) 


I get `false` returned - I'm a bit confused, do I need to do more after `api/claim` in order for that account to actually own the NFT, or am I maybe doing the signature wrong? 

Can you provide an example of successfully calling this function so I can understand what I'm doing wrong?

Reply:
If you're unsure about correct way to call some of the functions I would recommend checking the way it's done in `test.js` file, you can also run it using command `npm run test` to check the output and some of the logs.

In current implementation `/api/attendees` endpoint returns that user is one of attendees immediately after claim process was started. Meaning that the user requested claim and new NFT sell offer was created for him but it still has to be accepted by the user manually. 

On the other hand `/api/verifyOwnership?walletAddress` endpoint performs actual check using ledger data to see if user has NFT from selected event. That's why it returns `false` unless you accepted the NFT offer that was returned from `/api/claim` enpoint.

I hope that answers your question, this was conscious design decision I made to support different use cases but I am open to changes if you consider that to be important.